backend/lib/pipeline.ml

```diff
@@ -53,13 +53,13 @@ let dockerfile ~base =
 let weekly = Current_cache.Schedule.v ~valid_for:(Duration.of_day 1) ()
 
 (* $server/$repo_owner/$repo_name/pull/$pull_number *)
-let make_commit_status_url pull_number =
+let make_commit_status_url owner repository pull_number =
   let uri_end =
     match pull_number with
     | None -> ""
-    | Some number -> "/pull/" ^ string_of_int number
+    | Some number -> owner ^ "/" ^ repository ^ "/pulls/" ^ string_of_int number
```

Comment:
I just realised that I made a mistake and used `/pulls/n` instead of `/pull/n` in #41  (https://github.com/ocurrent/current-bench/pull/41/commits/1f751e119af19853be6736c84fa77d8e262fb258).

I prefer `/pull/n` because that's what GitHub uses.

Could you please change this?